sessions/function-development.qmd

```diff
@@ -28,13 +28,14 @@ package, documenting them, and how to stop your functions with an error.
 -   To use `@examples` to help with creating function?
 -   Function documentation (with roxygen2), part 1?
 
-**Session objectives**:
-
--   Convert a chunk of R code into a function
--   Include a function in an R package
--   Provide basic function documentation using Roxygen
--   Use functions from other packages in a function in your own package
--   Return an error message from a function
+**Learning objectives**:
+
+-   Convert your R code into a function
+-   Make your function modular and reusable
+-   Understand dependecy management and use functions from other
+    packages in your own package
```

Comment:
```suggestion
-   Explain what dependency means in the context of packages and how you formally manage dependencies when writing functions as part of a package
```

I'm not sure if maybe package dependency makes more sense in another session or here, but we'll keep it here for now.